includes/core-changes/aspnetcore/5.0/localization-requestlocalizationmiddleware-constructor-removed.md

```diff
@@ -0,0 +1,39 @@
+### Localization: Obsolete constructor removed in request localization middleware
+
+The <xref:Microsoft.AspNetCore.Localization.RequestLocalizationMiddleware> constructor that lacks an <xref:Microsoft.Extensions.Logging.ILoggerFactory> parameter was marked as obsolete [in this commit](https://github.com/dotnet/aspnetcore/commit/ba8c6ccf6fd3eeb7fc42a159d362b15eae4fb3a0). In ASP.NET Core 5.0, the obsolete constructor was removed. For discussion, see [dotnet/aspnetcore#23785](https://github.com/dotnet/aspnetcore/issues/23785).
+
+#### Version introduced
+
+5.0
```

Comment:
Don't you normally add the preview version too?

Reply:
I'm awaiting a response on that and will open a separate PR, if necessary.